src/lib/kdmp-parser-structs.h

```diff
@@ -90,8 +91,10 @@ static void DisplayField(const uint32_t Prefix, const char *FieldName,
       printf(": BMP Dump.\n");
       return;
     }
+    default:
+      printf(": Unknown.\n");
+      return;
```

Comment:
I had moved the `printf` statement out of the switch case on purpose - the compiler will warn you if you later add another enum value and don't add a switch case for it. If you have a `default` case then the compiler doesn't care because you have a fallback path.

Can we revert that part?

Reply:
Fixed with d6e988333dc7697797699e2f897f035245853028